Feisty.xcodeproj/project.pbxproj

```diff
@@ -19,6 +19,27 @@
 		5A6F57AA242CC7E7009190FA /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 5A6F57A9242CC7E7009190FA /* GoogleService-Info.plist */; };
```

Comment:
This file will show the addition of multiple .json files. Each file being a mock response for a game's details